src/ecdsa/test_ellipticcurve.py

```diff
@@ -80,7 +80,21 @@ def test_equality_curves(self):
     def test_inequality_curves(self):
         c192 = CurveFp(p, -3, b)
         self.assertNotEqual(self.c_23, c192)
-        
+
+    def test_usability_in_a_hashed_collection_curves(self):
+        {self.c_23: None}
+
+    def test_hashability_curves(self):
+        hash(self.c_23)
+
+    def test_conflation_curves(self):
+        ne1, ne2, ne3 = CurveFp(24, 1, 1), CurveFp(23, 2, 1), CurveFp(23, 1, 2)
+        eq1, eq2, eq3 = CurveFp(23, 1, 1), CurveFp(23, 1, 1), self.c_23
+        self.assertEqual(len({c_23, eq1, eq2, eq3}), 1)
+        self.assertEqual(len({c_23, ne1, ne2, ne3}), 4)
```

Comment:
`{}` for `set()` is not supported on Python2.6, code needs to be explicit: `set([c_23, ne1, ne2, ne3])`